Jellyfin.Plugin.AniDB/Providers/AniDB/Metadata/AniDbSeriesProvider.cs

```diff
@@ -32,6 +32,7 @@ public class AniDbSeriesProvider : IRemoteMetadataProvider<Series, SeriesInfo>,
         public static readonly RateLimiter RequestLimiter = new RateLimiter(TimeSpan.FromSeconds(3), TimeSpan.FromSeconds(5), TimeSpan.FromMinutes(5));
         private static readonly int[] IgnoredTagIds = { 6, 22, 23, 60, 128, 129, 185, 216, 242, 255, 268, 269, 289 };
         private static readonly Regex AniDbUrlRegex = new Regex(@"https?://anidb.net/\w+(/[0-9]+)? \[(?<name>[^\]]*)\]", RegexOptions.Compiled);
+        private static readonly Regex AniDbIdRegex = new Regex(@"\[anidb-([0-9]+)\]", RegexOptions.Compiled);
```

Comment:
I would suggest adding `id` as an optional part, and using named capture groups?
```suggestion
        private static readonly Regex AniDbIdRegex = new Regex(@"\[anidb(id)?-(?<anidb_id>[0-9]+)\]", RegexOptions.Compiled);
```

The matching code below would also need updating if using named capture groups.

Reply:
Done, thanks for the suggestion